Week3/js-exercises-week3/theLotteryMachine.js

```diff
@@ -0,0 +1,45 @@
+function sayThree() {
+    console.log("Say Three is called!!!!")
+}
+
+function sayFive() {
+    console.log("Say Five is called!!!!")
+}
+
+
+
+function threeFive(startIndex, stopIndex, sayThree, sayFive) {
+    const numbers = [];
+    // make array
+    for (let i = startIndex; i < stopIndex + 1; i++) {
+        numbers.push(i)
+
+    }
+
```

Comment:
too many spaces here. You can use prettier before committing the code
https://prettier.io/playground/